src/cogent3/maths/stats/distribution.py

```diff
@@ -72,20 +73,12 @@ def poisson_exact(successes, mean):
         return pdtrc(successes - 1, mean) - pdtrc(successes, mean)
 
 
+@c3warn.deprecated_callable(
+    version="2024.9", reason="use scipy.stats.binom.pmf()instead", is_discontinued=True
+)
 def binomial_exact(successes, trials, prob):
```

Comment:
The point here is not to replace our function with that from Scipy; rather, just use the decorator as indicated. Revert the change within the function to the original.

Reply:
Thanks for your reply! Does that mean what I'm going to do is keep the @c3warn.deprecated_callable message at the top, revert everything inside the function (including return), and then delete the new modules I imported?